packages/web/pages/stake/index.tsx

```diff
@@ -144,6 +141,12 @@ export const Staking: React.FC = observer(() => {
     return validatorSetPreferenceMap;
   }, [userValidatorPreferences]);
 
+  const validatorSquadModalAction: "stake" | "edit" = Boolean(
+    amountConfig.amount
```

Comment:
is `amountConfig.amount` a number or a string? Because if it's a string, '0' will evaluate as truthy